sdk/communication/communication-phone-numbers/test/public/siprouting/setRoutes.spec.ts

```diff
@@ -270,8 +287,17 @@ matrix([[true, false]], async function (useAad) {
       ];
       await client.setRoutes(routes);
 
-      assert.deepEqual(await client.getTrunks(), trunks);
-      assert.deepEqual(await client.getRoutes(), routes);
+      const getTrunks = await client.getTrunks();
+      const getRoutes = await client.getRoutes();
+
+      assert.isArray(getTrunks);
+      assert.isArray(getRoutes);
+      assert.equal(getTrunks.length, trunks.length);
```

Comment:
nit: can also use lengthOf:

```suggestion
      assert.lengthOf(getTrunks, trunks.length);
```